### DISCUSSION TITLE
Goals are listed in random order in the settings panel

### DISCUSSION THREAD

**Discussion by paulobarcelos** [plausible/analytics#2029] [Anything Else]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Using official Plausible Cloud hosting or self-hosting?

Plausible Cloud from plausible.io

### Describe the bug

On the Goals Settings page (https://plausible.io/xxxxxxxxx/settings/goals), the goals are listed in a seemly random order, which makes it hard to keep track of the goals, if you have many of them. I assume they should be listed in alphabetical order, or at least in the order they were created - but right now it doesn't do either.

### Expected behavior

Goals should be listed in alphabetical order

### Screenshots

<img width="873" alt="Screenshot 2022-07-18 at 16 31 10" src="https://user-images.githubusercontent.com/122277/179534747-98526f0a-328e-476a-8bc0-d79649931043.png">

### Environment

```markdown
- OS:
- Browser:
- Browser Version:
```

**Comment by metmarkosaric** [comment]
thanks for the feedback @paulobarcelos, makes sense to alphabetize this list or provide a search box! we keep "issues" for actual bugs in the product only so i have moved your idea to our feature request forum where we keep track of all the other feature requests. this way it's easier for our developers to keep track of your idea and prioritize it for the roadmap. thanks!

**Comment by jdestgermain** [comment]
Agree with this one!
